cli/cp.go

```diff
@@ -0,0 +1,291 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"io"
+	"strings"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
```

Comment:
We should improve the import sequence by https://github.com/alibaba/pouch/blob/master/docs/contributions/code_styles.md#rule003---import-packages